doc/source/ray-overview/use-cases.rst

```diff
@@ -96,11 +96,16 @@ Here, the focus is on training many models on subsets of a dataset. This is in c
 How do I do many model training on Ray?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-There are three ways of using Ray to express this workload.
+To train multiple independent models, use the Ray Tune (:ref:`Tutorial <mmt-tune>`) library. This is the recommended library for most cases.
 
-1. If you have a large amount of data, use Ray Data (:ref:`Tutorial <mmt-datasets>`).
-2. If you have a small amount of data (<10GB), want to integrate with tools such as Weights & Biases and MLflow, and have less than 20,000 models, use Ray Tune (:ref:`Tutorial <mmt-tune>`).
-3. If your use case does not fit in any of the above categories (e.g., you need to scale up to 1 million models), use Ray Core (:ref:`Tutorial <mmt-core>`), which gives you finer-grained control over the application. This option requires you to understand the Ray Core :ref:`design patterns and anti-patterns <core-patterns>`.
+You can use Tune with your current data preprocessing pipeline if your data source fits into the memory of a single machine (node). 
+If you need to scale your data, or you want to plan for future scaling, use the :ref:`Ray Data <datasets>` library with Ray Tune. 
```

Comment:
I don't think we have an example here of using Ray Data with Ray Tune (not sure if it would work anyways) -- it should only be with Ray data

Reply:
Ah. Good to know. Nice catch. TY